Yona/Yona/StringExtensions.swift

```diff
@@ -53,6 +53,19 @@ extension String {
         }
     }
     
+    func formatDutchCountryCodePrefix()->String{
+        var mobileNumber = self
+        if self.hasPrefix(YonaConstants.mobileDucthPrefix.generalPrefix) == true {
```

Comment:
No need to check `` == true``. Just ``if self.hasPrefix(YonaConstants.mobileDucthPrefix.generalPrefix) {`` should be sufficient.